warehouse/oidc/forms/_core.py

```diff
@@ -43,9 +43,11 @@ def validate_project_name(self, field):
             # not escaped by Jinja
             raise wtforms.validators.ValidationError(
                 markupsafe.Markup(
-                    f"This project already exists, use the project's publishing"
-                    f" settings <a href='{url}'>here</a> to create a Trusted"
-                    f" Publisher for it."
+                    _(
+                        "This project already exists, use the project's publishing"
+                        " settings <a href='{url}'>here</a> to create a Trusted"
+                        " Publisher for it.".format(url=url)
+                    )
```

Comment:
This isn't the way to handle parameters in translations, this needs to be something like:

```suggestion
                    _(
                        "This project already exists, use the project's publishing"
                        " settings <a href='${url}'>here</a> to create a Trusted"
                        " Publisher for it.",
                        mapping={"url": url},
                    )
```

Reply:
fixed!